subsys/bluetooth/host/Kconfig

```diff
@@ -1028,6 +1028,7 @@ config BT_CONN_DISABLE_SECURITY
 config BT_BREDR
 	bool "Bluetooth BR/EDR support [EXPERIMENTAL]"
 	depends on BT_HCI_HOST
+	select BT_CLASSIC
```

Comment:
Given that `BT_BREDR` is considered experimental, can't we just replace the name of the Kconfig? AFAIK we don't do deprecation of experimental things (even if it has been experimental for a long time...) 

If we do a simple rename, then we should put that in the migration guide for 3.7

Reply:
It is done. 

Could you guide me how to update the migration guide for 3.7?

Reply:
@lylezhu2012 It's basically just adding an entry in https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/releases/migration-guide-3.7.rst (maybe under a new section called Bluetooth Classic next to the Mesh and Audio sections) 

You can see the previous migrations guides at https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/releases/ to get an idea of what to write :)

Reply:
Got it. Thanks a lot.